core/common/src/main/java/alluxio/underfs/ObjectUnderFileSystem.java

```diff
@@ -495,8 +495,8 @@ public UfsDirectoryStatus getDirectoryStatus(String path) throws IOException {
       return new UfsDirectoryStatus(path, permissions.getOwner(), permissions.getGroup(),
           permissions.getMode());
     }
-    LOG.warn("Error fetching directory status, assuming directory {} does not exist", path);
-    throw new FileNotFoundException(path);
```

Comment:
Bad idea to through exception with only path. the error message eventually looks confusing, like this:

```
2021-02-10 02:34:30,893 DEBUG UnderFileSystemWithLogging - Exit (Error): GetStatus: path=s3://fluid/test/2020-02-05, Error=s3://fluid/test/2020-02-05
```